index.html

```diff
@@ -198,6 +201,7 @@
         const entries = await Promise.all(
           Array.from(listing, ({ suite, path, node }) => node.then(node => ({ suite, path, node })))
         );
+        // Show the list of suites in index.html? Right now it is empty and it isn't clear that you need to add stuff to the URL.
```

Comment:
It should probably just load cts by default. There's not much reason to mention the others, since unittests can run from node, and demos will probably be deleted.

Reply:
Sounds good, and done